Sources/BlueTriangle/Extensions/Bool+Utils.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Bool+Utils.swift
+//
+//  Created by Mathew Gacy on 2/17/22.
+//  Copyright © 2022 Blue Triangle. All rights reserved.
+//
+
+import Foundation
+
+extension Bool {
+    var smallInt: Int {
+        self ? 1 : 0
```

Comment:
```suggestion
        Int(truncating: self)
```

This would also work.